test/unit/views/DisassociateUcrSummaryViewSpec.scala

```diff
@@ -16,46 +16,38 @@
 
 package views
 
+import base.Injector
 import forms.DisassociateKind._
 import forms.DisassociateUcr
 import helpers.views.CommonMessages
-import play.twirl.api.Html
-import views.spec.UnitViewSpec
+import models.cache.DisassociateUcrAnswers
+import models.requests.JourneyRequest
+import play.api.mvc.AnyContentAsEmpty
+import views.html.disassociate_ucr_summary
 import views.tags.ViewTest
 
 @ViewTest
-class DisassociateUcrSummaryViewSpec extends UnitViewSpec with CommonMessages {
+class DisassociateUcrSummaryViewSpec extends ViewSpec with CommonMessages with Injector {
```

Comment:
We are moving away from using `CommonMessages`. It would be nice if you could remove it while you're here.

Reply:
Done.